tst/testinstall/bitfields.tst

```diff
@@ -0,0 +1,47 @@
+gap> START_TEST("bitfields.tst");
+
+# Test correct behaviour for a variety of  numbers of fields
+# and all getters and setters. 
+gap> for i in [1..2+GAPInfo.BytesPerVariable] do
+> bf := CallFuncList(MakeBitfields,[1..i]);
+> vals := List([1..i], j -> Random(0,2^j-1));
+> Add(vals, [1..i], 1);
+> x := CallFuncList(BuildBitfields,vals);
+> for j in [1..i] do
+> if bf.getters[j](x) <> vals[j+1] then
+> Print("Bad return from getter",i," ",j," ",x,"\n");
+> fi; od;
+> if bf.booleanGetters[1](x) <> (vals[2] = 1) then
+> Print("Bad return from Boolean Getter");
```

Comment:
Here it is "Boolean Getter", below it is "boolean setter" -- why the differing case?